tidb-cloud/import-with-mysql-cli.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Import Data into TiDB Cloud via MySQL Commands in Terminal
+summary: Learn how to import Data into TiDB Cloud via MySQL CLI.
+---
+
+# Import MySQL Data via MySQL Commands in Terminal
+
+## Prerequisites:
+
+Before you can import MySQL data to TiDB Cloud, you should have the following prerequisites in place:
+
+- Access to your TiDB Cloud Cluster.
+- Necessary connection details obtained from the TiDB Cloud portal.
+- An initial password created for your TiDB user (if not already done).
+- TiDB cluster connection information.
+
+## Steps:
+
+### 1. Connect to your TiDB Cloud Cluster
+
+1. Access the TiDB Cloud portal and obtain the necessary connection details.
```

Comment:
Maybe giving the audience instructions on getting connection details is a good idea. Here is an example that has already passed the review: https://github.com/pingcap/docs/blob/master/develop/dev-guide-gui-vscode-sqltools.md?plain=1#L44-L183